### PR TITLE
Update tmp_storage.rb to fix issue#11

### PR DESCRIPTION
**Pull request by Dvorf**
File open for token reading

### CODE REVIEW

File: lib/fnsapi/tmp_storage.rb [hunk 1]
```diff
@@ -13,6 +13,7 @@ def write_token(token, expire_at)
     end
 
     def token
+      @file = File.open(file_path, 'a+')
```
Comment (fedorkk) [inline lib/fnsapi/tmp_storage.rb:16]: We create the `@file` variable in object initializer. Why do you want to rewrite it here?

Comment (fedorkk) [reply lib/fnsapi/tmp_storage.rb:16]: https://github.com/actie/fnsapi/blob/master/lib/fnsapi/base_service.rb#L48 here we use the `token` method for the `TmpStorage` class instance, which initialized here: https://github.com/actie/fnsapi/blob/master/lib/fnsapi/base_service.rb#L41

So the `@file` variable should already exist for this instance.

**Comment by anaumov** [comment]
@Dvorf would you like to update PR?

**Comment by Dvorf** [comment]
Probably this will sound wired, but I was able to run fnsapi successfully only with my fix. Investigation showed that temporary token not being read from file.

**Comment by anaumov** [comment]
Closing this PR for now. Feel free to reopen it if needed.

**Pull request closed without merging**